src/templates/EventList.html

```diff
@@ -1,5 +1,11 @@
-<div id="event-list">
-  <div v-for="event in events" class="event-card">
-    {{event.title}}
+<div id="event-list" class="event-list" v-show="view === 'list'">
+  {{ filteredEvents.length ? '' : '' }}
```

Comment:
The stuff I've done here and in the event map html file is bananas -- I noticed that the components were not being updated when the store's values being piped into them changed, unless those values were actually referenced in the template. I just don't know Vue very well, there must be a real way to achieve this, anybody know?